analysis/performance_analysis/merged_performance_analysis.ipynb

```diff
@@ -1,29 +1,5 @@
 {
  "cells": [
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "**Table of contents**<a id='toc0_'></a>    \n",
-    "- [Normalizer Performance Analysis](#toc1_)    \n",
-    "  - [Import relevant packages](#toc1_1_)    \n",
-    "  - [Dictionaries to map variants to categories and record category counts](#toc1_2_)    \n",
-    "  - [CIViC](#toc1_3_)    \n",
-    "  - [MOA](#toc1_4_)    \n",
-    "  - [ClinVar](#toc1_5_)    \n",
-    "  - [Computing Coverage](#toc1_6_)    \n",
-    "  - [Generating Table](#toc1_7_)    \n",
-    "\n",
-    "<!-- vscode-jupyter-toc-config\n",
-    "\tnumbering=false\n",
-    "\tanchor=true\n",
-    "\tflat=false\n",
-    "\tminLevel=1\n",
-    "\tmaxLevel=6\n",
-    "\t/vscode-jupyter-toc-config -->\n",
-    "<!-- THIS CELL WILL BE REPLACED ON TOC UPDATE. DO NOT WRITE YOUR TEXT IN THIS CELL -->"
-   ]
-  },
```

Comment:
Why did you delete the table of contents?

Reply:
I am also wondering about that

Reply:
Oh did I?  Weird!

Reply:
Lemme re-insert it.